.travis.yml

```diff
@@ -7,24 +7,24 @@ matrix:
   fast_finish: true
   include:
     - os: linux
-      node_js: '7'
-      env: WEBPACK_VERSION="2.2.0" JOB_PART=lint
+      node_js: '8'
+      env: WEBPACK_VERSION="2.6.0" JOB_PART=lint
```

Comment:
Maybe we can add `WEBPACK_VERSION` env? This would make it convenient to update all versions